### PR TITLE
Added info about testing your changes to the Maji framework

### PR DESCRIPTION
**Pull request by RoelN**
Please review if this correct and clear enough.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,17 @@ To build a native app, run `bin/maji build <platform>`.
 
 For CI docker files are included, which can be used incombination with the [kabisa jenkins-docker tooling](https://github.com/kabisa/jenkins-docker). `dockerfiles` includes files for tests only or build android packages.
 
+## Developing Maji
```
Comment (eirvandelden) [inline README.md]: @RoelN Misschien hier van maken "Developing Maji and testing in a Maji App" ?

File: README.md [hunk 2]
```diff
@@ -82,6 +82,17 @@ To build a native app, run `bin/maji build <platform>`.
...
+
+When making changes to the Maji framework, you can test them by letting Node know you'll be using your local Maji version in a test app.
```
Comment (eirvandelden) [inline README.md]: "test them" > "test your changes in a Maji App"

File: README.md [hunk 3]
```diff
@@ -82,6 +82,17 @@ To build a native app, run `bin/maji build <platform>`.
...
+
+* Checkout Maji from Github
+* Make changes in Maji
+* Do a `npm link` in the Maji repo directory
+* Create a new project so you can test changes to the framework: `node lib/cli.js new org.example.mytestapp /path/to/mytestapp`
+* In the newly created project, link your local Maji: `npm link maji` and do a `bin/setup`
+* Do a `bin/setup` every time you want to test the changes in your Maji code
```
Comment (eirvandelden) [inline README.md]: (Only when changing the dependencies of Maji)

**Comment by eirvandelden** [comment]
@RoelN Kleine voorstel aanpassingen. Daarna: :shipit:

**Comment by RoelN** [comment]
These instructions don't seem to be correct as I still don't see my changes represented in the test app.

**Comment by eirvandelden** [comment]
Closing for now, until we figured out how best to test local changes :)

**Pull request closed without merging**